gpu_ocean/SWESimulators/CTCS.py

```diff
@@ -237,6 +237,9 @@ def step(self, t_end=0.0):
         Function which steps n timesteps
         """
         n = int(t_end / self.dt + 1)
+        if (n % 2 == 0):
+            n+=1
```

Comment:
This makes sure that we run an odd number of timestep, which is required to get the second order convergence ...